docs/topic/access-creds/cloud-auth.md

```diff
@@ -116,6 +116,20 @@ To do so, follow these steps:
    AWS accounts we create.
 4. Create the account! They'll receive an email with appropriate instructions.
 
+#### Gain root access to an AWS account
+
+In some cases we need to gain root access to an AWS account, e.g. migrating billing responsibility to a client. To do this,
+
+1. Log into the AWS SSO and take note of the Account ID (12 digits) and root user email address that you want to gain root access to. The root user email address is usually aliased to `support@2i2c.org`.
```

Comment:
Might be worth noting that this information can be found by viewing "Organizations" from the 2i2c-sandbox account https://us-east-1.console.aws.amazon.com/organizations/v2/home/accounts

Reply:
```suggestion
1. Log into the AWS SSO and take note of the Account ID (12 digits) and root user email address that you want to gain root access to. This information is also available by viewing "Organizations" from the [2i2c-sandbox account](https://us-east-1.console.aws.amazon.com/organizations/v2/home/accounts). The root user email address is usually aliased to `support@2i2c.org`.
```